src/System.Runtime/tests/System/Text/StringBuilderTests.netcoreapp.cs

```diff
@@ -199,5 +199,38 @@ public static void Insert_CharSpan_Invalid()
             AssertExtensions.Throws<ArgumentOutOfRangeException>("index", () => builder.Insert(builder.Length + 1, new ReadOnlySpan<char>(new char[0]))); // Index > builder.Length
             AssertExtensions.Throws<ArgumentOutOfRangeException>("requiredLength", () => builder.Insert(builder.Length, new ReadOnlySpan<char>(new char[1]))); // New length > builder.MaxCapacity
         }
+
+        public static IEnumerable<object[]> Equals_String_TestData()
```

Comment:
The tests for StringBuilder.Equals(STringBuilder) use this as a chunky one: `StringBuilder(20).Append(new string('a', 30));`